docs/dev/agent_integration.md

```diff
@@ -21,6 +22,52 @@ The required Datadog Agent integration development tools include:
 - [Docker][4] to run the full test suite.
 - The git [command-line][5] or [GitHub desktop client][19].
 
+You can list an Agent integration for a price in the Datadog Marketplace, or list it out-of-the-box on the Integrations page.
```

Comment:
Is it possible to make this section more detailed and actionable? 

For instance, after we explain the differences between the Marketplace and Integrations page options, can we add an imperative sentence that directs the user to "choose" one of the integration paths below using the buttons?

Reply:
@anawishnoff Moving this to the overview section might make it standout more and we can add actionable items.

Reply:
Yeah I agree with Alex, let's add this in after explaining the differences, and include a call to action (i.e. select an option below to get started)